### PR TITLE
Allow 0.25.x version of google-api-client

### PR DESCRIPTION
**Pull request by electron0zero**

### CODE REVIEW

File: google_drive.gemspec [hunk 1]
```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_dependency('nokogiri', ['>= 1.5.3', '< 2.0.0'])
-  s.add_dependency('google-api-client', ['>= 0.11.0', '< 0.22.0'])
+  s.add_dependency('google-api-client', ['>= 0.11.0', '< 0.25.0'])
```
Comment (gimite) [inline google_drive.gemspec]: You specify '< 0.25.0' here which means that 0.25.x is still not accepted. Is that OK?

**Review by gimite — CHANGES_REQUESTED** [review]
Thanks for the pull request.

File: google_drive.gemspec (hunk 1 above)
Comment (electron0zero) [reply google_drive.gemspec]: Yes, it works for me, 

on a side note 0.25.0 is the latest version  https://rubygems.org/gems/google-api-client/versions/0.25.0

Comment (electron0zero) [reply google_drive.gemspec]: I will change it to `=<` to be sure

Comment (electron0zero) [reply google_drive.gemspec]: changed it to `'google-api-client', ['>= 0.11.0', '< 0.25.1']` so people can get the latest version

Comment (electron0zero) [reply google_drive.gemspec]: it resolved to `google-api-client 0.25.0`

**Comment by electron0zero** [comment]
@gimite yeah makes sense, updated :raised_hands:

**Review by gimite — APPROVED** [review]

**Comment by gimite** [comment]
Thanks! Merged.

**Pull request merged**